### PR TITLE
Nonegatives, Nonpositives, Zeros with s

### PR DESCRIPTION
**Pull request by blegat**

### CODE REVIEW

File: src/attributes.jl [hunk 1]
```diff
@@ -94,7 +94,7 @@ Return a `Bool` indicating whether the solver instance `m` currently has a value
 ### Examples
 
 ```julia
-cangetattribute(GurobiSolver(), SupportsAffineConstraint{Zero}())
+cangetattribute(GurobiSolver(), SupportsAffineConstraint{Zeros}())
```
Comment (mlubin) [inline src/attributes.jl]: Could you remove this line? It's outdated

**Comment by mlubin** [comment]
And there's already a merge conflict

**Comment by mlubin** [comment]
I will hold off on merging anything else before this, to avoid more conflicts.

**Comment by blegat** [comment]
I will be afk until tomorrow, I will remove it later

**Comment by mlubin** [comment]
Ok, I'll update this PR

**Pull request merged**